src/simulator/SimulatorActionBar.tsx

```diff
@@ -56,27 +56,23 @@ const SimulatorActionBar = (props: SimulatorActionBarProps) => {
     }
     setIsMuted(!isMuted);
   }, [device, isMuted, setIsMuted]);
+  useEffect(() => {
+    device.on(EVENT_REQUEST_FLASH, handlePlay);
+    return () => {
+      device.removeListener(EVENT_REQUEST_FLASH, handlePlay);
+    };
+  }, [device, handlePlay]);
```

Comment:
We could consider doing this in simulator.ts but no strong view. Happy to merge as is.

Reply:
Rob pointed out that we don't have access to the file system from the device. Let's leave it as is.